init-service/src/main/java/uk/ac/ox/ndph/mts/init_service/model/Trial.java

```diff
@@ -0,0 +1,51 @@
+package uk.ac.ox.ndph.mts.init_service.model;
+
+import java.util.List;
+
+public class Trial {
```

Comment:
Maybe TrialConfiguration?

Reply:
a class named TrialConfiguration suggests (to me at least) that it is configuring a trial - this class is a 'dumb' domain object, I think it should be named after what it does... as it only holds Trial properties (fields) therefore I think its correctly named Trial ?

Reply:
Leave as Trial.